src/Microsoft.NET.Sdk.Razor/Sdk/Sdk.props

```diff
@@ -10,7 +10,5 @@ Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
 <Project ToolsVersion="14.0">
-
-  <Import Project="$(MSBuildThisFileDirectory)..\build\netstandard2.0\Sdk.Razor.CurrentVersion.props" />
-
+  <Import Project="$(MSBuildThisFileDirectory)..\build\netstandard2.0\Sdk.Razor.CurrentVersion.props" Condition="'$(_RazorSdkPackageReferenced)' != 'true'" />
```

Comment:
Re-importing this file twice was kinda fine in the past since we only set up `PropertyGroup` things. Now that we add to `ItemGroup` this becomes problematic since it results in duplicate entries.